src/controllers/payment.ts

```diff
@@ -120,11 +121,11 @@ export const sendPayment = async (req: Req, res: Res): Promise<void> => {
   // if remote text map, put that in
   let theChat = chat
   if (contact_ids) {
-    theChat = { ...chat.dataValues, contactIds: contact_ids } as Chat
+    theChat = <ChatRecord>{ ...chat.dataValues, contactIds: contact_ids }
```

Comment:
wondering why are we using ChatRecord and Partial<ChatPlusMembers> instead of just Chat, unless there's something I'm missing

Reply:
`chat` is a `ChatRecord` (returned by `helpers.findOrCreateChat`), then `theChat` is given the same type, but then assigned with `chat.dataValues`. it must be just a `Chat`

Reply:
fixed, and this code also looks better, less hacky type casts